cmd/server/main.go

```diff
@@ -2,17 +2,15 @@ package main
 
 import (
 	"context"
-	"errors"
 	"fmt"
 
 	"go.viam.com/rdk/services/web"
 	"go.viam.com/rplidar"
+	"go.viam.com/rplidar/cmd/helper"
```

Comment:
Generally "cmd" is reserved for "commands" or main programs, not a library you'd include from other code.